### ISSUE TITLE
no parameter `long paramsCheckIntervalMs`

### ISSUE THREAD

**Issue by yumemi-yli** [facebook/fresco#2351]
### Description

DefaultEncodedMemoryCacheParamsSupplier 

returns
```
return new MemoryCacheParams(
        maxCacheSize,
        MAX_CACHE_ENTRIES,
        maxCacheSize,
        MAX_EVICTION_QUEUE_ENTRIES,
        maxCacheEntrySize);
```

there is no parameter `long paramsCheckIntervalMs`

### Additional Information

* Fresco version: V1.13.0
* Platform version: sdk 28

**Comment by erikandre** [comment]
Hi @yumemi-yli 
Thanks for submitting the bug report. It seems like a straight forward change, if you need this urgently I recommend contributing a pull request with an implementation. Thanks!

**Comment by oprisnik** [comment]
This was a breaking change. I'll bring back the original constructor.

**Comment by oprisnik** [comment]
Fixed with deae4d34bcdbe392c77fb8e4529fcc0d3f51d699
